### PR TITLE
Clean up so i can deploy to heroku

### PR DESCRIPTION
**Pull request by styliii**
`heroku logs --tail` show some errors that i'm hoping these changes fix.
image

Made changes and ran `rails zeitwerk:check` until it was happy.

### CODE REVIEW

**Comment by styliii** [comment]
@keithtom going to merge this in, since it's only changing the stubbed out files.

File: app/models/s_s_j.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module SSJ
```
Comment (keithtom) [inline app/models/s_s_j.rb:1]: jsut FYI you should be able to write the file as ssj.rb

File: config/application.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ class Application < Rails::Application
     config.load_defaults 7.0
 
     config.autoload_paths += %W(#{config.root}/lib)
-    
+    config.eager_load_paths += %W(#{config.root}/lib)
```
Comment (keithtom) [inline config/application.rb:17]: why was this needed?

Comment (styliii) [reply config/application.rb:17]: not sure? `rails zeitwerk:check` suggested it done. Is it redundant with `autoload_paths`?

File: app/models/s_s_j.rb (hunk 1 above)
Comment (styliii) [reply app/models/s_s_j.rb:1]: image

Maybe `rails zeitwerk:check` is more fickle?

File: config/application.rb (hunk 2 above)
Comment (keithtom) [reply config/application.rb:17]: eager_load I believe forces all those files loaded on boot, autoload is a rails magic thing that does lazy class loading (it works by catching constant not defined errors and then trying to look for files matching the right naming conventing in all the autoload paths).  In production, you'd want to just load everything upfront, in development autoload keeps boot fast and only loads things as you need since you don't care about performance
